pages/quiz/index.js

```diff
@@ -0,0 +1,9 @@
+import QuizForm from "../../components/QuizCardPage/QuizForm";
+
+export default function Quiz() {
+  return (
+    <>
```

Comment:
You don't need the fragments here since you are returning one component only anyway 🙂